packages/rest/src/rest.server.ts

```diff
@@ -380,17 +370,11 @@ export class RestServer extends Context implements Server, HttpServerLike {
     return protocol + '://' + host;
   }
 
-  private async _redirectToSwaggerUI(
-    request: Request,
-    response: Response,
-    options: RestServerConfig,
-  ) {
+  private async _redirectToSwaggerUI(request: Request, response: Response) {
     const baseUrl =
-      options.apiExplorerUrl || 'https://loopback.io/api-explorer';
-    const openApiUrl = `${this._getUrlForClient(
-      request,
-      options,
-    )}/openapi.json`;
+      (this.config.apiExplorer && this.config.apiExplorer.url) ||
+      'https://loopback.io/api-explorer';
```

Comment:
Can you set the default value for `apiExplorer.url` inside `RestServer` constructor please? Not only it will simplify the code here, but it will make it easier for 3rd parties to introspect the API Explorer configuration and obtain the actual URL used even when no custom value was configured by the user.